pysatMissions/tests/test_instruments.py

```diff
@@ -83,6 +141,63 @@ def test_inst_cadence(self, inst_dict, kwarg, output):
             self.test_inst = pysat.Instrument(
                 inst_module=inst_dict['inst_module'])
 
-        self.test_inst.load(2019, 1)
+        self.test_inst.load(date=self.stime)
         cadence = np.diff(self.test_inst.data.index.to_pydatetime())
         assert np.all(cadence == dt.timedelta(seconds=output))
+        return
+
+    @pytest.mark.parametrize(
+        "kw_dict",
+        [{'one_orbit': True},
+         {'inclination': 13, 'alt_periapsis': 400, 'alt_apoapsis': 850,
+          'bstar': 0, 'arg_periapsis': 0., 'raan': 0., 'mean_anomaly': 0.},
+         {'TLE1': '1 25544U 98067A   18135.61844383  .00002728  00000-0  48567-4 0  9998',
```

Comment:
Do these "TLE" parameters need to be lower case as well?

Reply:
Key names, as strings, can be any case.  It's just the variable, function, and method names that should `always_be_snake_case`.

Reply:
But if they are changed already, it's fine.  Not a bad thing to be consistent with the variable names! 🐍

Reply:
Good catch.  Updating throughout.